alerter.cpp

```diff
@@ -21,11 +28,13 @@ void alertInCelcius(float farenheit) {
         // Add a test below to catch this bug. Alter the stub above, if needed.
         alertFailureCount += 0;
     }
+		
 }
 
 int main() {
     alertInCelcius(400.5);
     alertInCelcius(303.6);
+	assert(alertFailureCount ==1);

```

Comment:
This test will pass when either one of the above increments the `alertFailureCount`. Strengthen the test to make it clear which one should alert and which one shouldn't